### PR TITLE
Added 80legs

### PR DESCRIPTION
**Pull request by brentclark**
Good day

Please consider my PR for 80legs.
You can get more information from 
http://80legs.com/
https://en.wikipedia.org/wiki/80legs

Kind Regards
Brent Clark

### CODE REVIEW

**Comment by brentclark** [comment]
I later added sysscan
https://www.cyber.nj.gov/threat-profiles/trojan-variants/sysscan

Kind Regards
Brent Clark

File: rules/scanners-user-agents.data [hunk 1]
```diff
@@ -142,6 +142,9 @@ sqlmap
 # sql injection
 # http://sqlninja.sourceforge.net/
 sqlninja
+#sysscan
```
Comment (fgsch) [inline rules/scanners-user-agents.data]: IMO you can drop the comments since it's the same.

Comment (csanders-git) [reply rules/scanners-user-agents.data]: In fact, iit seems like these are used in the rest of the file to indicate the typee of scanner.

Comment (brentclark) [reply rules/scanners-user-agents.data]: @fgsch done !!

Comment (fgsch) [reply rules/scanners-user-agents.data]: What @csanders-git said. 
My original comment (and @csanders-git one) also applies to 80legs.

**Comment by csanders-git** [comment]
Thanks for the commit!!! @brentclark congrats!

**Pull request merged**